WoWsShipBuilder.Web/Dialogs/BuildImageRenderDialog.razor

```diff
@@ -0,0 +1,340 @@
+                    }
+                    @if (twoColumns)
+                    {
+                        <MudStack Row Spacing="2">
+                            <MudStack Spacing="2" Style="width: 50%">
+                                @if (showCaptainSkills)
+                                {
+                                    <CaptainSkillSelector ViewModel="@CaptainSkillSelectorViewModel" ShipClass="@Ship.ShipClass" IsBuildImage/>
+                                }
+                                @if (showUpgrades)
+                                {
+                                    <ShipUpgradeSelector ViewModel="@UpgradePanelViewModel" IsBuildImage/>
+                                }
+                            </MudStack>
+                            <MudStack Spacing="2" Style="width: 50%">
+                                @if (showSignals)
+                                {
+                                    <div style="@((showConsumables ? "height: 83.13%;" : "height: 100%;") + "display: flex")">
+                                        <SignalSelector ViewModel="@SignalSelectorViewModel" IsBuildImage/>
+                                    </div>
+                                }
+                                @if (showConsumables)
+                                {
+                                    <div style="height: 16.87%; display: flex">
+                                        <ConsumableSelector ViewModel="@ConsumableViewModel" IsBuildImage/>
+                                    </div>
+                                }
+                            </MudStack>
+                        </MudStack>
+                    }
+                    else
+                    {
+                        if (showCaptainSkills)
+                        {
+                            <CaptainSkillSelector ViewModel="@CaptainSkillSelectorViewModel" ShipClass="@Ship.ShipClass" IsBuildImage/>
+                        }
+                        if (showUpgrades)
+                        {
+                            <ShipUpgradeSelector ViewModel="@UpgradePanelViewModel" IsBuildImage/>
+                        }
+                        if (showConsumables)
+                        {
+                            <ConsumableSelector ViewModel="@ConsumableViewModel" IsBuildImage/>
+                        }
+                        if (showSignals)
+                        {
+                            <SignalSelector ViewModel="@SignalSelectorViewModel" IsBuildImage/>
+                        }
+                    }
+                </MudStack>
+            </div>
+        </MudBreakpointProvider>
+    </DialogContent>
+    <DialogActions>
+        <MudButton OnClick="Cancel" Color="Color.Error">
+            @Localizer.GetAppLocalization(nameof(Translation.Cancel)).Localization
+        </MudButton>
+        <MudButton OnClick="@OnConfirm" Color="Color.Info">
+            Get Url
```

Comment:
Also, better separation between the image download and the build link buttons would be nice. Maybe use some spacing?

Reply:
Can't add any more spacing between buttons otherwise they get messed up in the single column version of the dialog